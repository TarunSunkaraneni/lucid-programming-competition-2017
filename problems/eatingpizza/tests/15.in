10
2 5 5
10 1 1
10 2 5
9 2 4
5 1 3
4 1 5
2 4 5
6 3 3
10 1 3
8 4 5
