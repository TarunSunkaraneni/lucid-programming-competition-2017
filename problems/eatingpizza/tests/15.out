Jacob
Jacob
Megan
Megan
Megan
Megan
Jacob
Jacob
Megan
Megan
